Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_madgraphMLM-pythia8/TTto2L2Nu-3Jets_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,24 @@
+{
+    "template": "LO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "ickkw": "1",
+        "maxjetflavor": "5",
+        "xqcut" : "20"
+    },
+    "model_params": "massW.dat",
+    "model_params_user": [
+    ],
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer.dat", "PartonShower/madgraphMLM-pythia8.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'JetMatching:qCut = 70.'",
+            "'JetMatching:nQmatch = 5'",
+            "'JetMatching:nJetMax = 3'",
+            "'TimeShower:mMaxGamma = 4.0'"
```

Comment:
We leave this at 4 GeV as we are providing ttZ in 1-4 GeV and 4-10 GeV? 
20/70 are the xqcut and qcut previously used ?

Reply:
Ah sorry I forgot about this myself. We should use 1.0GeV this time and then move up to 4.0 from 2023 campaign